browser/ui/views/brave_actions/brave_rewards_action_view.h

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_ACTIONS_BRAVE_REWARDS_ACTION_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_ACTIONS_BRAVE_REWARDS_ACTION_VIEW_H_
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/memory/raw_ptr.h"
+#include "base/memory/weak_ptr.h"
+#include "brave/browser/brave_rewards/rewards_panel_coordinator.h"
+#include "brave/browser/brave_rewards/rewards_tab_helper.h"
+#include "brave/components/brave_rewards/browser/rewards_notification_service.h"
+#include "brave/components/brave_rewards/browser/rewards_notification_service_observer.h"
+#include "brave/components/brave_rewards/browser/rewards_service.h"
+#include "brave/components/brave_rewards/browser/rewards_service_observer.h"
+#include "chrome/browser/ui/tabs/tab_strip_model_observer.h"
+#include "chrome/browser/ui/views/toolbar/toolbar_button.h"
+#include "components/prefs/pref_change_registrar.h"
+#include "ui/gfx/geometry/skia_conversions.h"
+#include "ui/views/widget/widget_observer.h"
+
+class Profile;
+class TabStripModel;
+
+template <typename T>
+class WebUIBubbleManagerT;
+
+class RewardsPanelUI;
+
+// A button that lives in the actions container and opens the Rewards panel. The
+// button has an associated context menu and can be hidden by user settings.
+class BraveRewardsActionView
+    : public ToolbarButton,
```

Comment:
Just wondering why we're using `ToolbarButton` and not `LabelButton` like our other buttons in the location bar?

Reply:
I got your answer over DM - it's for the menu

Reply:
Exactly!